app/models/authorization.rb

```diff
@@ -39,6 +39,10 @@ def self.omniauth_image(auth)
     end
   end
 
+  def self.external_providers
+    return self.where('provider.ne': 'tomatoes')
```

Comment:
What would you think of a scope instead of a class method? The result would be very similar, but I like scopes because they're easier to spot when you search in the code. See https://docs.mongodb.com/ruby-driver/master/tutorials/5.1.0/mongoid-queries/#scoping.